rbc/remotejit.py

```diff
@@ -14,10 +14,7 @@
 from .thrift import Server, Dispatcher, dispatchermethod, Data, Client
 from .utils import get_local_ip, UNSPECIFIED
 from .targetinfo import TargetInfo
-# XXX WIP: the OmnisciCompilerPipeline is no longer omnisci-specific because
-# we support Arrays even without omnisci, so it must be renamed and moved
-# somewhere elsef
```

Comment:
This comment is still valid, isn't it? I suggest creating an issue before deleting this comment.

Reply:
It is not valid anymore. The pipeline is heavydb specific and non-heavydb code was removed in https://github.com/xnd-project/rbc/pull/458. The idea of supporting arrays outside heavydb was to tackle the memory management issue. Since this problem is being tackled in https://github.com/heavyai/heavydb-internal/pull/6424, I've removed the code from RBC.